src/components/PokemonDisplay.js

```diff
@@ -0,0 +1,107 @@
+import React, {setState, useEffect} from 'react';
+import $ from 'jquery'
+import Button from '@material-ui/core/Button'
+
+class PokemonDisplay extends React.Component {
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            index: 0,
+            value: "",
+            pokemon: {
+                name:				"",
+                nationalPokedexNum: -1,
+                artworkUrl:			"",
+                type:				"",
+                species:			"",
+                height:				"",
+                weight:				""
+            }
+        }
+        this.CreatePokemonObject = this.CreatePokemonObject.bind(this);
+        this.GetWebsiteHTML = this.GetWebsiteHTML.bind(this);
+        this.StartScrape = this.StartScrape.bind(this);
+        this.componentDidUpdate = this.componentDidUpdate.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
+    }
+
+    async GetWebsiteHTML(webUrl)
+     {
+        var result = await $.ajax({ 
+            url: webUrl,
+            statusCode: {
+                404: function() {
+                    console.log("URL returned 404, does the pokemon/website exist?");
+                }
+            },
+            success: function(data) {  result = data; },
+            error: function(xhr) { console.log('Request Status: ' + xhr.status + ' Status Text: ' + xhr.statusText + ' ' + xhr.responseText); },
+        });
+         return result;
+     }
+
+    
+    CreatePokemonObject(pageSource)
+     {
+         var divHtml = $('<div>', {html:pageSource});
+         var tableHtml = $('<table>', {html:pageSource});
+         var tableStr = tableHtml.find('.vitals-table');
+    
+        var loadedPokemon = {
```

Comment:
Probably want to make a pokemon data parsing method and have it handle negative cases and returning appropriate errors if the pageSource html changes in the future

Reply:
For the moment, added some simple error handling.